vendor/engines/projects/lib/projects/engine.rb

```diff
@@ -22,6 +22,10 @@ class Engine < Rails::Engine
                         "after_order_status",
                         "projects/shared/select_project"
 
+      ViewHook.add_hook "order_management.order_details.print_details",
```

Comment:
This actually applies to the view hook above this, but since the partial the view hook is in changed, it needs to change. Now, we're not showing the project on the form anymore.